fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -1534,6 +1536,8 @@ private void handleInsertStmt() throws Exception {
         }
 
         analyzeVariablesInStmt(insertStmt.getQueryStmt());
+        // reset the executionTimeout corresponding with the StmtExecutor
+        context.setExecTimeout(Math.max(context.getSessionVariable().getInsertTimeoutS(), context.getExecTimeout()));
```

Comment:
Therefore we don't need a reset here.

Reply:
still need here, since hints in select will change the default insert_timeout